governance.md

```diff
@@ -0,0 +1,164 @@
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Op1st GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a subproject | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
+| Subproject owner | Set direction and priorities for a subproject | Demonstrated responsibility and excellent technical judgement for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+### Community groups
+The Op1st community is initiating Governance 1.0 with only one type of community group, Special Interest Groups (SIGs).
+The purpose of doing so is to empower SIGs to gather interested people around a subject area, then decide and implement that into governance.
+For example, one of the first acts of the Community SIG is to establish a milestone schedule for defining and holding elections for future steering committee(s).
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
```

Comment:
Is any member able to open a SIG? When we say "Anyone" here ... what does that actually mean WRT a SIG?

Reply:
> Is any member able to open a SIG? When we say "Anyone" here ... what does that actually mean WRT a SIG?

I've been mulling on this one, as I wasn't sure if a clarification of meaning was needed, or have you hit upon something that needs a group consensus. I think it's both. :)

1. IMO, any community member (as defined in that section) should be empowered to propose a SIG, according to whatever criteria the Community SIG defines (X type of mission, Y number of members, etc.) But I don't think it should be easy or encouraged for people to join the project with the singular goal of opening a new SIG (unless invited by the community to do so.) That is, I see SIGs arising from community-identified needs, so an outsider would need to sync with the community enough to "be one of us" in order to create consensus around the need.
2. I do think it's OK and encouraged to join the project specifically to join a SIG. In that case, the language here is intended to remind people that participating in a SIG is a way of participating in the project overall regardless of interest and focus. Even a one-off contribution process should have the clear expectation that a one-off contributor is following the Code of Conduct. So the language here is to make it clear that one is welcome to drop into a SIG, offer a patch, and disappear forever ... but it all has to be done under the same aegis as a regular, repeat contributor

Reply:
The work in https://github.com/operate-first/community/commit/d1adfd2e8848392404e1a9052e9e8156dbfdc1a8 covers the SIG lifecycle. I believe this provides enough of an answer to this question. The remainder can be decided by the community and not set in stone here.